cupy/testing/array.py

```diff
@@ -90,6 +92,32 @@ def assert_array_equal(x, y, err_msg='', verbose=True):
         cupy.asnumpy(x), cupy.asnumpy(y), err_msg=err_msg,
         verbose=verbose)
 
+    if strides_check:
+        if x.strides != y.strides:
+            msg = ["Strides are not equal:"]
+            if err_msg:
+                msg = [msg[0] + ' ' + err_msg]
+            if verbose:
+                msg.append(" x: {}".format(x.strides))
+                msg.append(" y: {}".format(y.strides))
+            raise AssertionError('\n'.join(msg))
+        if x.flags.c_contiguous != y.flags.c_contiguous:
```

Comment:
I think this is redundant. `flags.c_contiguous` and `flags.f_contiguous` are just reduced representations of `strides`.